meson.build

```diff
@@ -638,7 +638,8 @@ endif
 
 ############################################################
 
-if run_command('python3', '-c', 'import jinja2', check : false).returncode() != 0
+python = find_program('python3')
+if run_command(python, '-c', 'import jinja2', check : false).returncode() != 0
```

Comment:
This is semantically identical, since the first thing run_command does with the initial argument is to internally run the find_program() implementation on it.

Reply:
The documentation doesn't mention this and to be honest I haven't looked at the source, but if that's true it doesn't respect the configured python3 from a native file.